src/alerts/model.rs

```diff
@@ -1,9 +1,14 @@
 use chrono::{DateTime, Utc};
-use serde::{Serialize, Deserialize};
+use serde::{Deserialize, Serialize};
 
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct WebhookNotification {
     pub message: String,
     pub probe_name: String,
     pub failure_timestamp: DateTime<Utc>,
-}
\ No newline at end of file
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct SlackNotification {
+    pub text: String,
+}
```

Comment:
struct is destined to be a POST body, format from https://api.slack.com/messaging/webhooks#posting_with_webhooks